### ISSUE TITLE
Copy all non-Markdown files directly to the build.

### ISSUE THREAD

**Issue by NickSto** [galaxyproject/galaxy-hub#883]
Currently we copy all files with a [hardcoded list](https://github.com/galaxyproject/galaxy-hub/blob/master/src/build/run.mjs#L64) of file extensions to the build. This is so they can be directly linked to.

Instead of enumerating all the extensions to include we could use a [negated glob](https://github.com/sindresorhus/globby#globbing-patterns) to only exclude `.md` (I think it'd be `"!**/*.md"`).

FYI, here are how many files we currently have of every other extension:
```
$ find content -type f -not -iname '*.md' | awkt '{n = split($0,fields,"."); print fields[n]}' | tr '[:upper:]' '[:lower:]' | sort | uniq -c | sort -rg -k 1
   2752 png
    504 jpg
     55 gif
     39 svg
     18 jpeg
     10 pdf
      3 sh
      3 scss
      2 txt
      2 plist
      1 xcf
      1 key
      1 1
```

**Comment by dannon** [comment]
I generally lean towards explicit definitions of what's staged and what isn't, but it's worth looking at what the bottom half of that list actually represents in the repo though.  I do see that we have actual `jpeg` extension files, though, which is not currently included in the explicit list, so I'll add that.

**Comment by NickSto** [comment]
Yeah, I can see the sense in not going with a default "allow all". And it's true that you can always put something in `/static/`, which will always work.

I guess then authors would just have to know that if they want to share a random file they have to put it in `/static/`. Might be a little potential confusion if they're used to linking directly to images, but then they try a non-image file (or even an unusual image format like a `.webp`) and it doesn't work.

**Comment by NickSto** [comment]
Btw here's all the files from the bottom part of that list:
```
$ find content -type f -not -iname '*.md' | grep -v '^content/images/' | awkt '{n = split($0,fields,"."); print fields[n]}' | tr '[:upper:]' '[:lower:]' | sort | uniq -c | sort -rg -k 1 | tail -n 7 | awk '{print $2}' | while read ext; do find content -type f -iname "*.$ext"; done
content/tutorials/nt_rnaseq/get_img.sh
content/tutorials/var_dip/cp_img.sh
content/tutorials/var_hap/cp_img.sh
content/style/styles.scss
content/style/variables.scss
content/style/overrides.scss
content/tutorials/datasets/gemini_tables.txt
content/tutorials/datasets/variant_impacts.txt
content/cloudman/getting-started/cm-getting-started.graffle/data.plist
content/cloud/jetstream/storage/jetstream-storage.graffle/data.plist
content/outreach/powered-by-galaxy/PoweredByGalaxy.xcf
content/tutorials/images/RNAseq_histiry_image.key
content/tutorials/collections/collectionCreation.png.1
```
I do remember seeing `PoweredByGalaxy.xcf` before. This was in a [list](https://galaxyproject.org/outreach/powered-by-galaxy/) of different Galaxy logos people could use - this was the GIMP-formatted version. I think most of the others (e.g. the `.sh`, `.scss`, and `.plist`) aren't actually used (linked to), though.
